doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| FOP field instance                                      | A data value in a fop instance, corresponding to a fop field in instance's format is called a fop field instance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Fop State Machine (fom)                                 | fom is a state machine that represents current state of the fop's [r.fop] ST execution on a node. fom is associated with the particular fop and implicitly includes this fop as part of its state                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
FOM is a state machine that represents current state of the fop's execution on a node. FOM is associated with the particular FOP and implicitly includes this FOP as part of its state.